cmd/wallet-web/src/router/index.js

```diff
@@ -71,7 +71,30 @@ router.beforeEach(async (to, from) => {
       console.error('error authenticating user: invalid hash received');
       return false;
     }
-  } else if (to.matched.some((record) => record.meta.requiresAuth)) {
+  }
+
+  if (to.matched.some((record) => record.meta.requiresCHAPI)) {
+    // Get enableCHAPI from wallet config
+    if (store.getters.getEnableCHAPI === undefined) {
+      if (store.dispatch('loadUser') && store.getters.getEnableCHAPI === undefined) {
+        await store.dispatch('initOpts');
+      }
+    }
+
+    if (!store.getters.getEnableCHAPI) {
+      store.dispatch('updateUserLoaded', true);
+      return {
+        name: 'NotFound',
+        params: {
+          title:
+            'The page you requested requires CHAPI, but your Wallet configuration does not support it at the moment.',
+          message: 'Please, contact your administrator.',
```

Comment:
There are no French string provided for the `Not Found` page as of now since we don't have the designs. As discussed, this is a temporary page to be replaced/updated when the designs are provided